app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ApplicationRecord
+
```

Comment:
If we're going to format things like this we should go through and make sure it is all the same... just FYI